docs/UserGuide.md

```diff
@@ -49,18 +49,20 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
 
 ## Features
 
-<box type="info" seamless>**Notes about the command format:**<br>
+<box type="info"><md>**Notes about the command format:**</md><br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+  e.g `n/NAME [owe/OWED_AMOUNT]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
```

Comment:
hmm, t/TAG is removed be the examples not updated yet! 😢 
> `n/John Doe t/friend` or as `n/John Doe`.

These are still using PREFIX_TAG, right?